integrations/iceberg-views/src/main/java/org/apache/iceberg/nessie/NessieExtCatalog.java

```diff
@@ -24,9 +24,9 @@
 import org.apache.iceberg.TableOperations;
 import org.apache.iceberg.catalog.Namespace;
 import org.apache.iceberg.catalog.TableIdentifier;
-import org.apache.iceberg.view.View;
-import org.apache.iceberg.view.ViewDefinition;
-import org.apache.iceberg.view.Views;
+import org.apache.iceberg.viewdeprecated.View;
```

Comment:
Maybe ` org.projectnessie.viewpoc.View`? Having `deprecated` in the package name does not look nice and will probably be ignored anyway. WDYT?

Maybe add `@Deprecated` to the class?